.github/workflows/containers.yml

```diff
@@ -116,7 +116,7 @@ jobs:
 
             -   name: Run CI Tests
                 run: >
-                    ./gradlew :integration-tests:runOauth2Tests --info -Denvironment.config=-docker -Denvironment.offPlatform=true -Dokta.client.id=${{ secrets.OKTA_CLIENT_ID }} -Dokta.client.password=${{ secrets.OKTA_CLIENT_PASSWORD }}
```

Comment:
could the task itself be renamed as well to runOidcTests?

Reply:
Yes, lets make it consistent now.